global_planner/src/planner_core.cpp

```diff
@@ -166,6 +166,9 @@ void GlobalPlanner::reconfigureCB(global_planner::GlobalPlannerConfig& config, u
     planner_->setNeutralCost(config.neutral_cost);
     planner_->setFactor(config.cost_factor);
     publish_potential_ = config.publish_potential;
+    show_footprint_radii_ = config.show_footprint_radii;
+    if (!show_footprint_radii_)
+      clearFootprintRadii();
```

Comment:
S: I don't think we need this. They're also not clearing the published potential anyway. If you really want this maybe just set the lifetime of the markers to 1.0s so that it auto clears if you don't publish a new one. That way we don't need this extra function.

Reply:
That's why I suggested using the lifetime field. Would solve that problem without the need to create a new function. But this is fine too

Reply:
> S: This is fine, but now that I think about it -- why are we publishing this in the global planner instead of the costmap itself?? That would also make it available for the local costmap footprint if we ever need it.

good point,,, I put here cause we needed to decide the padding for the global costmap,,, but I agree, makes more sense on costmap_2d!